assets/wizards/popups/components/prompt-action-card/index.js

```diff
@@ -22,58 +23,175 @@ import './style.scss';
-						tooltipPosition="bottom center"
-					/>
-					<Button
-						isQuaternary
-						isSmall
-						className={ popoverVisibility && 'popover-active' }
-						onClick={ () => setPopoverVisibility( ! popoverVisibility ) }
-						icon={ moreVertical }
-						label={ __( 'More options', 'newspack' ) }
-						tooltipPosition="bottom center"
-					/>
-					{ popoverVisibility && (
-						<PrimaryPromptPopover
-							onFocusOutside={ () => setPopoverVisibility( false ) }
-							prompt={ prompt }
-							{ ...props }
+		<>
+			<ActionCard
+				isSmall
+				badge={ placementForPopup( prompt ) }
+				className={ className }
+				title={ title.length ? decodeEntities( title ) : __( '(no title)', 'newspack' ) }
+				titleLink={ decodeEntities( editLink ) }
+				key={ id }
+				description={ description }
+				notification={ warning }
+				notificationLevel="error"
+				actionText={
+					<Fragment>
+						<Button
+							isQuaternary
+							isSmall
+							className={ categoriesVisibility && 'popover-active' }
+							onClick={ () => setCategoriesVisibility( ! categoriesVisibility ) }
+							icon={ settings }
+							label={ __( 'Category filtering and campaigns', 'newspack' ) }
+							tooltipPosition="bottom center"
 						/>
-					) }
-					{ categoriesVisibility && (
-						<SecondaryPromptPopover
-							onFocusOutside={ () => setCategoriesVisibility( false ) }
-							prompt={ prompt }
-							segments={ segments }
-							{ ...props }
+						<Button
+							isQuaternary
+							isSmall
+							className={ popoverVisibility && 'popover-active' }
+							onClick={ () => setPopoverVisibility( ! popoverVisibility ) }
+							icon={ moreVertical }
+							label={ __( 'More options', 'newspack' ) }
+							tooltipPosition="bottom center"
 						/>
+						{ popoverVisibility && (
+							<PrimaryPromptPopover
+								onFocusOutside={ () => setPopoverVisibility( false ) }
+								prompt={ prompt }
+								setModalVisible={ setModalVisible }
+								{ ...props }
+							/>
+						) }
+						{ categoriesVisibility && (
+							<SecondaryPromptPopover
+								onFocusOutside={ () => setCategoriesVisibility( false ) }
+								prompt={ prompt }
+								segments={ segments }
+								{ ...props }
+							/>
+						) }
+					</Fragment>
+				}
+			/>
+			{ modalVisible && (
+				<Modal
+					className="newspack-popups__duplicate-modal"
+					title={ sprintf( __( 'Duplicate “%s”', 'newspack' ), title ) }
+					onRequestClose={ () => {
+						setModalVisible( false );
+						setDuplicateTitle( null );
+						resetDuplicated();
+					} }
+				>
+					{ duplicated ? (
+						<>
+							<Notice
+								isSuccess
+								noticeText={ sprintf(
+									__( 'Duplicate of “%s” created as a draft.', 'newspack' ),
+									title
+								) }
+							/>
+							{ ! campaignGroups && (
+								<Notice
+									isWarning
+									noticeText={ __( 'The prompt is currently unassigned.', 'newspack' ) }
```

Comment:
I'd suggest the same wording changes as in the [other review](https://github.com/Automattic/newspack-popups/pull/564#pullrequestreview-694723438).